.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
   - docker
 branches:
   only:
+    - main
```

Comment:
This legacy workflow could run on `main` but will only deploy when run on `master`. That's potentially inconsistent, but I don't think it is a problem.  As you note, this is a legacy workflow.